pytorch/generate_ssh_keys.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright (c) 2023 Intel Corporation
```

Comment:
If this is the first time, this file is introduced here, we want `2024`

Reply:
We might get rid of this file here as Tyler mentioned earlier it will be there in this [PR ](https://github.com/intel/ai-containers/pull/124/files)

Reply:
sure.